### ISSUE TITLE
Unable to open <filepath>: File is a directory

### ISSUE THREAD

**Issue by Emerentius** [Microsoft/pyright#64]
**Describe the bug**
Under windows 10 with the VSCode Extension, the links pyright puts under `Problems` don't work and show the titular error. The error message *does* show the full path however.

If I right-click and select `copy` on one of the problems and paste the json into a text-editor, I can see that the resource field points to `/`. Under linux the extension works and `resource` contains the full and correct path.

```
{
	"resource": "/",
	"owner": "_generated_diagnostic_collection_name_#0",
	"severity": 8,
	"message": "Import 'matplotlib.pyplot' could not be resolved",
	"source": "pyright",
	"startLineNumber": 414,
	"startColumn": 16,
	"endLineNumber": 414,
	"endColumn": 33
}
```

**To Reproduce**
Open a python file with issues and click on one of the problems to be redirected to the location.

**Expected behavior**
Redirect to the problem location.

**VS Code extension or command-line**
VSCode 1.33 with Pyright v1.0.9

**Additional context**
Windows 10 Pro 1803
Python 3.7.1 from Anaconda3

**Comment by erictraut** [comment]
Thanks for the bug report. I haven't done any of the Pyright development on Windows, and this was clearly a case I hadn't tested. I found the problem, and it will be fixed in the next release.

**Comment by erictraut** [comment]
This is fixed in version 1.0.10, which I just published. If you still see the problem, please reopen the bug report.

**Comment by Anon1777** [comment]
I am having this issue too, was your problem fixed and if so, how did you fix it?
